.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Rust CI"
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: Build contracts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: nightly-2023-04-17
+      - run: yarn
+      - run: yarn install
```

Comment:
`yarn` with no command runs `yarn install` by default, therefore line 14 and 15 does the same thing, there is no need for both. See [yarn cli documentation](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-default-command) for details.